test-resources/override.js

```diff
@@ -1,5 +1,6 @@
 var Mocks = require('../target/mocks/mocks.js');
 var Module = require('module');
+process.env.TZ = 'Etc/Universal'
```

Comment:
`Etc/UTC` better if there's is, `Etc/Universal` is the old name

Reply:
it does : https://nodejs.org/api/cli.html#tz:~:text=such%20as%20%27-,Etc/UTC,-%27%2C%20%27Europe
updated